packages/console/src/components/Topbar/logo.svg

```diff
@@ -0,0 +1,58 @@
+<svg width="891" height="274" viewBox="0 0 891 274" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
I would recommend putting these svg files in a dedicated assets folder, say `src/assets/images`, instead of scattered around in each component folder

Reply:
make sense for logo